4_MT1D_5layer_inversion.ipynb

```diff
@@ -0,0 +1,1134 @@
+{
```

Comment:
@sgkang: did you change anything in this notebook? It was renamed earlier to 3_MT1D_5layer_inversion, so I think this is a duplicate